hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -143,7 +143,7 @@ public class HoodieWriteConfig extends HoodieConfig {
 
   public static final ConfigProperty<Integer> WRITE_TABLE_VERSION = ConfigProperty
       .key("hoodie.write.table.version")
-      .defaultValue(HoodieTableVersion.current().versionCode())
+      .defaultValue(HoodieTableVersion.SIX.versionCode())
```

Comment:
oh, now I see why CI is failing.

Lets keep this as 8. But only in those 15 to 20 odd tests classes, we can add option to override "hoodie.write.table.version"

So, that we can still expect a green CI for rest of test classes.